### DISCUSSION TITLE
Updating pull request via Rest API works in my own private repository only

### DISCUSSION THREAD

**Discussion by benoittgt** [community/community#38163] [Enterprise]
Hello

I have one repository in my company organisation. I have a token with those scopes
```
["notifications",
 "read:discussion",
 "read:enterprise",
 "read:gpg_key",
 "read:org",
 "read:packages",
 "read:public_key",
 "read:repo_hook",
 "read:user",
 "repo",
 "user:email",
 "workflow"]
```
I have one private repostitory in own organisation with a token those scopes
```
["repo", "workflow"]
```

I am able to update the pull request on my repository but not on the repository in my company.

```
PATCH https://api.github.com/repos/<owner>/<repository>/pulls/9: 404 - Not Found // See: https://docs.github.com/rest/reference/pulls#update-a-pull-request (Octokit::NotFound)
```

**Comment by david-engelmann** [comment]
Are the public repositories working with protected branches?

**Comment by benoittgt** [comment]
Yes. But I removed the protected branch for `master`, retried the call but still have the same error. 

I tried to GET the pull request but without any issue, only PATCH is failing.
